experimental/dds/tree/src/Transaction.ts

```diff
@@ -254,6 +254,8 @@ export class Transaction {
 		const node = this.view.getSnapshotNode(change.nodeToModify);
 		const { payload } = change;
 		const newNode = { ...node };
+		// Rationale: 'null' is used as a jsonable sentinel for the empty payload.
```

Comment:
I think we should phrase this differently. SharedTree explicitly differentiates between an empty payload (a payload of 0 length) and no payload. In this case null is not being used to indicate an empty payload, but to indicate no payload.

The reason null is used here, instead of undefined is a bit questionable (its keeping open the door for some future compatibility where we also allow undefined which would mean something else). There is some documentation on the subject on SetValue.payload. Referencing that might be a good choice here.

Reply:
I found the comment, thx!